OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/Optimove.java

```diff
@@ -552,6 +559,17 @@ public void trackEddystoneBeaconProximity(@NonNull String hexNamespace, @NonNull
         Optimobile.trackEddystoneBeaconProximity(context, hexNamespace, hexInstance, distanceMetres);
     }
 
+    /**
+     * Records a proximity event for an iBeacon
+     * @param uuid
+     * @param majorId
+     * @param minorId
+     * @param proximity - Constant that reflects the relative distance to a beacon.
+     */
+    public void trackIBeaconProximity(@NonNull String uuid, @NonNull String majorId, @NonNull String minorId, @Nullable IBeaconProximity proximity) {
```

Comment:
Are the major/minors not ints for ibeacons? (maybe I'm wrong)

Reply:
(https://developer.apple.com/documentation/corelocation/clbeacon/1621418-major)

Reply:
OK so it's spec'd as NSNumber, which I suppose could contain floats in theory. The iBeacon guide describes it as a 2-byte value for both major/minor so perhaps precision would be practically quite limited.

What do you think?